help/develop/javascript-development/datagrok-configuration.md

```diff
@@ -0,0 +1,39 @@
+# Datagrok configuration
+
+After you install `datagrok-tools` and run the first command (probably, to [generate a package]), you need to configure
+your environment.
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Developer Keys](#developer-keys)
+
+## Overview
+
+The Grok CLI creates a configuration file under `%USERNAME%/.grok/config.yaml` to store the configurations for your
```

Comment:
We try not to abbreviate the platform name (or is it just me?). Let's consult @skalkin and @dskatov. I would say that it's okay when talking about Grok script. When I see this shortening in other contexts, I typically replace it. 

Regarding the config file path, it would be nice to give more details. We choose the home directory via this command: https://nodejs.org/api/os.html#os_os_homedir. It relies on environment variables, and there are some OS-specific differences:

- Windows: `C:\Users\%Username%\.grok\config.yaml`
- Linux: `~/.grok/config.yaml`